test/services/easypost.test.js

```diff
@@ -1,14 +1,148 @@
 import { expect } from 'chai';
 
-import EasyPost from '../../src/easypost';
+import EasyPost, { METHODS } from '../../src/easypost';
 import MissingParameterError from '../../src/errors/general/missing_parameter_error';
+import Fixture from '../helpers/fixture';
+import * as setupPolly from '../helpers/setup_polly';
 
 /* eslint-disable func-names */
 describe('EasyPost', function () {
+  setupPolly.startPolly();
+
   it('throws an error when no API key is provided', async function () {
     expect(() => new EasyPost()).to.throw(
       MissingParameterError,
       'Missing required parameter: API Key.',
     );
   });
+
+  it('will log the appropriate values when a request and response hooks are provided', async function () {
+    let requestConfig;
+    const requestHook = (response) => (requestConfig = response);
+    let responseConfig;
+    const responseHook = (response) => (responseConfig = response);
+
+    const client = new EasyPost(process.env.EASYPOST_TEST_API_KEY);
```

Comment:
Nit: instead of defining `client` every time in each unit test, you can define it globally. Example: https://github.com/EasyPost/easypost-node/blob/master/test/services/address.test.js#L14-L16

Reply:
👍